src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -453,7 +454,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
                 if (containsEmoji(content, effect.emojis)) {
                     // For initial threads launch, chat effects are disabled
                     // see #19731
-                    const isNotThread = this.props.relation?.rel_type !== RelationType.Thread;
+                    const isNotThread = this.props.relation?.rel_type !== THREAD_RELATION_TYPE.name;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?